x230/README.md

```diff
@@ -161,57 +167,62 @@ based supply for a second USB port (like [this](https://de.aliexpress.com/item/1
 
 ![ch341a programmer with extra USB power supply](ch341a.jpg)
 
-#### unpack the Skulls release archive
+### Side note
+Connecting an ethernet cable as a power-source for SPI (instead of the VCC pin)
+is not necessary (some other flashing how-to guides mention this).
+Setting a fixed (and low) SPI speed for flashrom offeres the same stability.
+Our scripts do this for you.
 
+I checked around on excactly when and how one should connect the clip. There has been no clear advice, it seemed to be safest to first connect all cables between the flashing device and the clip, and once correctly configured, connect the clip to the chip.
 
-	tar -xf skulls-x230-<version>.tar.xz
-	cd skulls-x230-<version>
+## Get and unpack the Skulls release archive
+After choosing one flasher-option [download](https://github.com/merge/skulls/releases) the latest release and untar it:
 
 
-#### ifd unlock and me_cleaner: the 8MB chip
-The [Intel Management Engine](https://en.wikipedia.org/wiki/Intel_Management_Engine)
-resides on the 8MB chip (at the bottom, closer to you).
-We don't need to touch it for coreboot-upgrades in the future, but to
-enable internal flashing, we need to unlock it once, and remove the Management
-Engine for
-[security reasons](https://en.wikipedia.org/wiki/Intel_Management_Engine#Security_vulnerabilities):
+	tar -xf skulls-x230-<version>.tar.xz
+	cd skulls-x230-<version>
 
+Make sure to verify the checksum with e.g.:
 
-	sudo ./external_install_bottom.sh -m -k <backup-file-to-create>
+	sha256sum skulls-x230-0.1.0.tar.xz anc compare this to the [checksum](https://github.com/merge/skulls/releases/download/0.1.0/skulls-x230-0.1.0.tar.xz.sha256)
 
+### Side note
+Flashing with these low speeds takes time. Be patient. E.g. unlocking the bottom chip with its two reads, one write and one verify step usually takes one hour in total. Again, be patient!
 
-That's it. Keep the backup safe.
+## First, optional step: Connect to the bottom chip
+There are a few reasons why you may start with connecting your clip to the bottom (at the bottom, closer to you) chip (it has the same pinout than the upper chip):
+- You may want to enable in system updates in the future. The advantage is that you can update and change whatever you decide to flash in the upper chip. The disadvantage is that any software can flash you BIOS with this setting. Choose wisely (Heads - see below - may be of use here).
```

Comment:
that might be useful actually. documenting this chip to be optional might be a good idea...